interpreter/binary/decode.ml

```diff
@@ -135,6 +135,12 @@ let sized f s =
 
 open Types
 
+let var_type s =
+  let pos = pos s in
+  match vs33 s with
+  | i when i >= 0l -> SynVar i
```

Comment:
I was confused about this until I realised it reads `zero L` and not `zero one` :-)